src/sub-blocks/PriceCard/schema.ts

```diff
@@ -1,11 +1,9 @@
-import {
-    AnimatableProps,
-    BaseProps,
-    ButtonBlock,
-    CardBase,
-    LinkProps,
-    MediaProps,
-} from '../../schema/validators/common';
+import pick from 'lodash/pick';
+
+import {BaseProps, ButtonBlock, CardBase, LinkProps} from '../../schema/validators/common';
+import {ContentBase} from '../Content/schema';
+
+const PriceCardContentProps = pick(ContentBase, ['theme']);
```

Comment:
is there only one property we pick? mb jus add like
`
...
theme: ContentBase.theme
...
`?